build/files.js

```diff
@@ -231,6 +231,22 @@ function CorkedRequest(state) {
       /Buffer\.prototype\.copy\.call\(src, target, offset\);/
     , 'src.copy(target, offset);'
   ]
+  , maybeDefineProperty = [
+        headRegexp
+      , '$1\n' + `
+/* <replacement> */
+var maybeDefineProperty = Object.defineProperty;
+// IE < 9
+if (!maybeDefineProperty || global.document && global.document.documentMode < 9) {
```

Comment:
@ScottFreeCode you might be more familiar with IE 8 than me, can you confirm this is the correct way to detect if we are running on IE8? I need to disable it even though it is present.